lib/zipkin-tracer/zipkin_json_tracer.rb

```diff
@@ -14,6 +14,10 @@ def perform(json_api_host, spans)
       req.headers['Content-Type'] = 'application/json'
       req.body = JSON.generate(spans_with_ips)
     end
+  rescue Net::ReadTimeout, Faraday::ConnectionFailed => e
+    error_message = "Got #{e.class.inspect} with Message #{e.message} error while connecting to #{json_api_host}. " \
+                    "- Please make sure the Zipkin server is running in that url and port."
```

Comment:
I think "running in that URL and port" is not accurate
What about "... running at the specified location" or, .... nothing? I think "error while connecting to {host}" is explicit enough.

@ssteeg-mdsol ideas?

Reply:
How about
```
"Error while connecting to #{json_api_host}: #{e.class.inspect} with message '#{e.message}'. " \
Please make sure the URL / port are properly specified for the Zipkin server
```